editor/index.html

```diff
@@ -3,9 +3,8 @@
 	<head>
 		<title>three.js / editor</title>
 		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
```

Comment:
Ops! Seems like you deleted the viewport meta tag.

Reply:
Sry! Should be fine now.